server/py/services/alerts/crud/alerts.py

```diff
@@ -178,9 +178,27 @@ def delete_alert(
             )
 
         framework.utils.singletons.db.get_db().delete_alert(session, project, name)
-        self._clear_alert_states(alert)
+        self._clear_alert_states(alert.id)
         self._clear_caches(alert.id)
 
+    def delete_alerts(
+        self,
+        session: sqlalchemy.orm.Session,
+        project: str,
+    ):
+        project = project or mlrun.mlconf.default_project
+        services.alerts.crud.Events().delete_project_alert_events(project)
+
+        alert_ids = framework.utils.singletons.db.get_db().delete_project_alerts(
+            session, project
+        )
+        if not alert_ids:
+            return
+
+        for alert_id in alert_ids:
```

Comment:
worth adding log for before operations (and after all). include information for debugging but make sure it is not spammy.
```suggestion
        project = project or mlrun.mlconf.default_project
        logger.debug("deleting project alert events", project=project, alert_name=name)
        services.alerts.crud.Events().delete_project_alert_events(project)
        logger.debug("deleting project alert", project=project)
        alert_ids = framework.utils.singletons.db.get_db().delete_project_alerts(
            session, project
        )
        if not alert_ids:
            return
        logger.debug("deleting project alert ids", alert_ids_length=len(alert_ids))
        for alert_id in alert_ids:
          ...
        logger.debug("successfully deleted project alert", project=project, alert_name=name)
        
```